### ISSUE TITLE
Warning when used within a form

### ISSUE THREAD

**Issue by szehao** [ooade/react-click-away-listener#29]
![image](https://user-images.githubusercontent.com/32298647/106129895-a8844d00-619b-11eb-9c4f-0371b5306fe0.png)

Building a custom select dropdown with ClickAwayListener wrapping the component.

Right now the structure looks something like this
```
<form onSubmit={e => e.preventDefault()}>
       <ClickAwayListener>
               <button onClick={e => e.preventDefault()}> test </button>
       </ClickAwayListener>
       <button type='submit'>submit</button>
</form>
```

**Comment by ooade** [comment]
Please, what version of are you using?

**Comment by szehao** [comment]
Version 1.5.1

**Comment by ooade** [comment]
Hi, it has been fixed in 1.5.2 🙏
